packages/compiler/package.json

```diff
@@ -66,7 +66,7 @@
     "mkdirp": "~1.0.4",
```

Comment:
Add a comment here that the production playground is broken and needs to be resolved before merging.

Reply:
I guess this is resolved.